docs/_basic/listening_events.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Listening to events
+lang: en
+slug: event-listening
+order: 3
+---
+
+<div class="section-content">
+
+You can listen to [any Events API event](https://api.slack.com/events) using the `event()` method after subscribing to it in your app configuration. This allows your app to take action when something happens in Slack, like a user reacting to a message or joining a channel.
```

Comment:
```suggestion
You can listen to [any Events API event](https://api.slack.com/events) using the `event()` method after subscribing to it in your app configuration. This allows your app to take action when something happens in a workspace it's installed, like a user reacting to a message or joining a channel.
```

Reply:
Optional suggestion. This just aligns it with some changes we've made in the getting started guide recently. If you think it adds confusion, just leave this suggestion out.